src/mscorlib/shared/System/IO/Path.Windows.cs

```diff
@@ -119,14 +119,21 @@ public static bool IsPathRooted(string path)
         {
             if (path != null)
             {
-                int length = path.Length;
-                if ((length >= 1 && PathInternal.IsDirectorySeparator(path[0])) ||
-                    (length >= 2 && PathInternal.IsValidDriveChar(path[0]) && path[1] == PathInternal.VolumeSeparatorChar))
-                    return true;
+                return IsPathRooted(path.AsReadOnlySpan());
```

Comment:
Nit, but I'd make this `return path == null ? false : IsPathRooted(path.AsReadOnlySpan());`